packages/ember-routing/lib/services/router.js

```diff
@@ -62,8 +62,24 @@ const RouterService = Service.extend({
        attempted transition
      @public
    */
-  replaceWith() {
+  replaceWith(/* routeNameOrUrl, ...models, options */) {
     return this.router.replaceWith(...arguments);
+  },
+
+  /**
+     Generate a URL based on the supplied route name.
+
+     @method urlFor
+     @param {String} routeName the name of the route
+     @param {...Object} models the model(s) or identifier(s) to be used while
+       transitioning to the route.
+     @param {Object} [options] optional hash with a queryParams property
```

Comment:
My interpretation was that we should mimic the signatures of `transitionTo` and `replaceWith`, as it's preferable to have a consistent API across all public methods. That said, I think what you said here is totally legit, in that the `queryParams` object should *be* the queryParams itself, not an object having a `queryParams` property that is the actual queryParams. 

I'm going to revisit all of them for consistency, but I know that `EmberRouter` expects an object that has a `queryParams` property that it unpacks. If we go this route we may want to revisit how `EmberRouter` is handling extracting the QPs.

Thoughts?